src/services/table-details.ts

```diff
@@ -385,6 +385,29 @@ export const TEST_RESULT_TABLE: TableDetails = {
     ]
 };
 
+export const ACTIVITY_TABLE: TableDetails = {
+    tableName: "activity",
+    columnNames: [
+        "test_station_id",
+        "tester_id",
+        "activityId",
+        "parentId",
+        "activityType",
+        "startTime",
+        "activityDay",
+        "endTime",
+        "notes"
+    ]
+};
+
+export const WAIT_REASON_TABLE: TableDetails = {
+    tableName: "wait_reason",
+    columnNames: [
+        "activity_id",
+        "note"
+    ]
+};
+
 export const allTables = (): TableDetails[] => {
```

Comment:
I am not sure if is used anywhere, should it contain WAIT_REASON_TABLE as well ?

Reply:
Added wait reason table to this list